packages/drivers/odsp-driver/src/odspDriverUrlResolverForShareLink.ts

```diff
@@ -200,6 +200,12 @@ export class OdspDriverUrlResolverForShareLink implements IUrlResolver {
         const shareLink = await this.getShareLinkPromise(odspResolvedUrl);
         const shareLinkUrl = new URL(shareLink);
 
+        let actualDataStorePath = dataStorePath;
+        // If the user has passed an empty dataStorePath, then extract it from the resolved url.
```

Comment:
Add description of this behavior into param comments so it is clear to the caller what to expect in case of passing empty string. This is change in behavior so important to document publicly.